BootloaderCorePkg/Tools/PrepareBuildComponentBin.py

```diff
@@ -237,13 +239,11 @@ def Main():
     # Leave the final path node as '$AUTO' to allow to determine the repo dir automatically.
     ucode_repo_dir = os.path.abspath (os.path.join(workspace_dir, '$AUTO'))
 
-    fsp_inf = os.path.join(sbl_dir, 'Silicon', silicon_pkg_name, 'FspBin', 'FspBin.inf')
```

Comment:
if sbl_dir is not SblOpen folder, please double check if it still work. I guess this line is needed.
 fsp_inf  = os.path.join(sbl_dir, fsp_inf)
same to the microcode_inf

Reply:
There is no problem finding the files.

Reply:
ok. thanks for the tests.